Sources/Basic/Path.swift

```diff
@@ -485,6 +499,16 @@ extension AbsolutePath {
     public func contains(_ other: AbsolutePath) -> Bool {
         return self.components.starts(with: other.components)
     }
+
+    /// Checks the path and throws a `PathError` if path is invalid.
```

Comment:
Can you create a new PR for this change? We usually create a new PR for changes that can be landed independently.